script/lookup-commit.sh

```diff
@@ -73,7 +73,7 @@ update_gitgit_dir () {
 		git -C "$GITGIT_DIR" fetch maintainer ||
 		die "Could not update the 'maintainer' remote to $GITGIT_DIR"
 	else
-		git -C "$GITGIT_DIR" remote add -f maintainer https://github.com/gitster/git ||
+		git -C "$GITGIT_DIR" remote add -f maintainer https://github.com/ttaylorr/git ||
```

Comment:
@ttaylorr On its own, this would not have the intended effect because it only initializes the remote if it does not exist yet.

Since you _also_ want to change the nickname of the remote, it would work this time.

But once Junio takes back the reigns, it would no longer be as easy as reverting this change because then the `maintainer` remote is already registered and will pull from the wrong fork!

A better idea would probably be to update this script (which _is_ used in https://dev.azure.com/gitgitgadget/git/_build?definitionId=9) to install a `ttaylorr` remote and (temporarily) use that one, but keep the name `gitster` for the existing functionality and use it again after the revert.

However, I think you missed https://github.com/gitgitgadget/gitgitgadget/blob/609d70813de217a8385e7405eb49f92481885d0c/script/lookup-commit.sh#L247-L251 which _does_ need to be updated.

FWIW this seems to be [the only instance where GitGitGadget's source code mentions the word "interim"](https://github.com/search?q=repo%3Agitgitgadget%2Fgitgitgadget%20interim&type=code).